### PR TITLE
Added schedule input, input parsing, and reformatted code into objects

### PR DESCRIPTION
**Pull request by BradleyCai**
Added ability to input growl schedule and parse it as well as reformatted all of the code to be more object oriented. There's also a lot of bug fixes, design changes, and regex changes. The regexes have been updated to deal with more edge cases.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -31,16 +31,26 @@ a:focus,
...
 }
 
 div.centered table {
-    margin: 0 auto; 
+    margin: 0 auto;
     text-align: left;
 }
 .table-space {
-    margin-bottom: 25px;
+    /*margin-bottom: 0px;*/
```
Comment (emmiegit) [inline css/style.css:53]: Why are you commenting out this line as opposed to removing it?

File: regex/noncap-building.regex [hunk 2]
```diff
@@ -1 +1 @@
-(?:[A-Z\-]*)
+(?:[A-Z\-]{0,8})
```
Comment (emmiegit) [inline regex/noncap-building.regex:1]: I'm assuming you choose 8 as the maximum length based on the building codes I entered, right?

File: regex/noncap-hours.regex [hunk 3]
```diff
@@ -1 +1 @@
-(?:[0-9]{4}[AP]M)?-([0-9]{4}[AP]M)?
+(?:[0-9]{4}[AP]M)?-?(?:[0-9]{4}[AP]M)?
```
Comment (emmiegit) [inline regex/noncap-hours.regex:1]: Why is the dash that separates the two sections optional? Is `1130AM0200PM` a valid time?

File: regex/noncap-building.regex (hunk 2 above)
Comment (BradleyCai) [inline regex/noncap-building.regex:1]: Yes, otherwise it picks up the next class as the name of the building

File: regex/noncap-hours.regex (hunk 3 above)
Comment (BradleyCai) [inline regex/noncap-hours.regex:1]: I didn't see the "?" outside of the group. Sorry.

File: css/style.css (hunk 1 above)
Comment (BradleyCai) [inline css/style.css:53]: Some left over commenting, my bad.

**Pull request merged**